src/components/Search.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import "../styles/search.css";
+import getImages from "../requests/getImages";
+
+const Search = ({ setSearchResults }) => {
+  const [value, setValue] = useState();
```

Comment:
```
  const [searchTerm, setSearchTerm] = useState();
```
value is a very generic name. Maybe use searchTerm instead. Also consider adding a default value of an empty string seeing as we expect this to be a string when the user types in the input field